classes/form/CustomerForm.php

```diff
@@ -127,6 +127,19 @@ public function validate()
             $birthdayField->setValue($dateBuilt->format('Y-m-d'));
         }
 
+        // password check
+        $passwordValue = $this->getValue('password');
+        if (!empty($passwordValue) && !Validate::isPasswd($passwordValue)) {
+            $passwordField = $this->getField('password');
+            $passwordField->addError(
+                $this->translator->trans(
+                    'At least 8 characters long, 1 lowercase letter, 1 uppercase letter, 1 digit and 1 special char',
```

Comment:
```suggestion
                    'At least %number% characters long, 1 lowercase letter, 1 uppercase letter, 1 digit and 1 symbol (eg. @!?^&)',
```

Shouldn't this message start with something like: "Your password should be at least... and contain at least..."?